### DISCUSSION TITLE
how to run dev without wrangler

### DISCUSSION THREAD

**Discussion by QMangoJ** [flareact/flareact#253] [General]
how to run dev without wrangler?

![Screen Shot 2022-11-12 at 10 04 00 PM](https://user-images.githubusercontent.com/35953248/201477715-08d9a5d5-8be6-41c4-a1e9-60aab0291ed3.png)

**Comment by tj-kev** [comment]
Does your ```wrangler.toml``` file include the line ```type = "webpack"```? This is required for v1 of wrangler.
